pkg/apis/picchu/v1alpha1/revision_types.go

```diff
@@ -72,6 +72,7 @@ type RevisionSpec struct {
 	TrafficPolicy    *istiov1alpha3.TrafficPolicy `json:"trafficPolicy,omitempty"`
 	Failed           bool                         `json:"failed"`
 	IgnoreSLOs       bool                         `json:"ignoreSLOs,omitempty"`
+	CanaryWithSLIRules bool                         `json:"canaryWithSLIRules,omitempty"`
```

Comment:
looks like this hasn’t been `go fmt`ted

Reply:
Weird, my vscode didn't format this one for some reason. Fixed!